hapic/doc.py

```diff
@@ -14,42 +14,48 @@
 from hapic.description import ControllerDescription
 
 
+def generate_schema_ref(spec, schema):
```

Comment:
inkhey, could you add typing on the function signature?

Reply:
done.